storm-core/src/jvm/org/apache/storm/trident/Stream.java

```diff
@@ -124,6 +124,31 @@ public Stream parallelismHint(int hint) {
     }
 
     /**
+     * Sets the CPU Load resource for the current node
+     */
+    public Stream setCPULoad(Number load) {
```

Comment:
It is better to have these configs to be set at respective operation level in a stream with intuitive API than setting at topology level. Below API is suggested which may look more verbose than setXXXLoad method but it clearly conveys what it is for.   
But we should not have API for this configuration at topology level as it requires to name these operations and use those names with load configs in topology looks to be clunky. 

``` java
TridentTopology topo = new TridentTopology();
TridentState wordCounts = 
    topology.newStream("spout1", spout)
        .withLoadConfig(new LoadConfigBuilder().
                        .withCPULoad(50)
                        .withMemoryLoad(1024).build())
        .each(new Fields("sentence"), new Split(), new Fields("word"))
         .withLoadConfig(new LoadConfigBuilder().
                        .withCPULoad(20)
                        .withMemoryLoad(256).build())
        .groupBy(new Fields("word"))
        .persistentAggregate(new MemorymapState.Factory(), new Count(), new Fields("count"))
        .parallelismHint(6);
```